addons/website/views/snippets/s_table_of_content.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_table_of_content" name="Table of Content">
+    <section class="s_table_of_content pt24 pb24">
+        <div class="container table_of_content">
+            <div class="row s_col_no_resize s_nb_column_fixed">
+                <div class="col-lg-3 s_table_of_content_navbar_wrap s_table_of_content_navbar_sticky
+                        s_table_of_content_vertical_navbar d-print-none d-none d-lg-block o_not_editable bg-white"
+                        data-name="Navbar">
+                    <div class="s_table_of_content_navbar list-group"/>
+                </div>
+                <div class="col-lg-9 s_table_of_content_main oe_structure oe_empty" data-name="Content">
+                    <section>
+                        <div class="container">
+                            <div class="row">
+                                <div class="col-lg-12 pb16">
+                                    <h1 class="text-secondary" data-anchor="true">Intuitive system</h1>
+                                    <h4>What you see is what you get</h4>
+                                    <p>
+                                        Insert text styles like headers, bold, italic, lists and fonts with
+                                        a simple WYSIWYG editor. Flexible and easy to use.
+                                    </p>
+                                    <h4>Customization tool</h4>
+                                    <p>
+                                        Click and change content directly from the front-end: no complex back
+                                        end to deal with.
+                                    </p>
+                                    <h4>Building blocks system</h4>
+                                    <p>
+                                        Create your page from scratch by dragging and dropping pre-made,
+                                        fully customizable building blocks.
+                                    </p>
+                                </div>
+                                <div class="col-lg-12 pb16">
+                                    <h1 class="text-secondary" data-anchor="true">Design features</h1>
+                                    <h4>Bootstrap based templates</h4>
+                                    <p>
+                                        Easily design your own Odoo templates thanks to clean HTML
```

Comment:
Did not we agree on two section with no column ?

Reply:
It was not clear (SBU spec, FP spec, etc.). But I agree that it is cleaner without columns so I remove them